### PR TITLE
Add notification element

### PR DESCRIPTION
**Pull request by DDHLong**
In this PR, I add layout for the notification for notification RecycleView 

Reference #9

### CODE REVIEW

File: app/src/main/res/layout/fragment_notification.xml [hunk 1]
```diff
@@ -1,18 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
-<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    tools:context=".NotificationFragment"
-    android:background="#121212">
+    android:background="#121212"
```
Comment (dinhanhx) [inline app/src/main/res/layout/fragment_notification.xml]: Question: will this value be used again somewhere else?

**Comment by dinhanhx** [comment]
@DDHLong Please reference this PR to correct issue

File: app/src/main/res/layout/fragment_notification.xml (hunk 1 above)
Comment (DDHLong) [reply app/src/main/res/layout/fragment_notification.xml]: i dont remember deleted this? hmmmm

**Comment by DDHLong** [comment]
> @DDHLong Please reference this PR to correct issue

hmmm I think we should devise the issue smaller this PR cannot solve #9

**Comment by dinhanhx** [comment]
@DDHLong I agree with you. You divide the issue into smaller one then link pr #9 to the corresponding one. I say "references" which mean it's related to that issue not to solve it

**Review by dinhanhx — APPROVED** [review]

**Pull request merged**